src/TestSuite/Constraint/Console/ContentsContain.php

```diff
@@ -40,6 +40,6 @@ public function matches($other): bool
      */
     public function toString(): string
     {
-        return sprintf('is in %s,' . PHP_EOL . 'actual result:' . PHP_EOL . $this->contents, $this->output);
+        return sprintf('is in %s,' . PHP_EOL . 'actual result:' . PHP_EOL, $this->output) . $this->contents;
```

Comment:
Can we encode the newline with double quotes? `"is in %s,\n..."` ?

Reply:
That aint better, given that the constant is Platform awake imo.

Reply:
If it needs the windows-specific end lines, then no worries.